tests/integration/test_ai_optimizations.py

```
@@ -0,0 +1,196 @@
+import unittest
+import threading
```

Ruff (0.8.2)

`threading` imported but unused

Remove unused import: `threading`

(F401)